onnxruntime/test/mlas/unittest.cpp

```diff
@@ -3198,7 +3198,7 @@ class MlasQuantizeLinearTest : public MlasTestBase
 
         for (size_t n = 0; n < N; n++) {
             if (Output[n] != OutputReference[n]) {
-                printf("exp difference: size=%u, index=%u, output=%d, expected=%d\n", unsigned(N), unsigned(n), Output[n], OutputReference[n]);
+              printf("exp difference: size=%u, index=%u, output=%d, expected=%d\n", unsigned(N), unsigned(n), int(Output[n]), int(OutputReference[n]));
```

Comment:
nit: can you change this back to 4 spaces style?

Reply:
Sure.